spec/u-namespace.md

```diff
@@ -67,6 +72,8 @@ not valid, or some other reason.
 
 ### `u:dir`
 
+Implementations SHOULD support this option.
```

Comment:
I don't think we can or should make any `u:` option MUST in this release, but really everyone should implement this feature. Otherwise string metadata about direction has nowhere to go. For those playing at home, my [UTW presentation](https://www.youtube.com/watch?v=eJEKbUx3B0w) explains why.